satellite/metainfo/metainfo.go

```diff
@@ -536,22 +556,34 @@ func (endpoint *Endpoint) checkBucketPointers(ctx context.Context, req *pb.SetAt
 		Time:          time.Now(),
 	})
 	if err != nil {
-		return false, status.Errorf(codes.Unauthenticated, err.Error())
+		return nil, status.Errorf(codes.Unauthenticated, err.Error())
 	}
 
 	prefix, err := CreatePath(ctx, keyInfo.ProjectID, -1, req.BucketName, []byte(""))
 	if err != nil {
-		return false, err
+		return nil, err
 	}
 
 	items, _, err := endpoint.metainfo.List(ctx, prefix, string(""), string(""), true, 1, 0)
 	if err != nil {
-		return false, err
+		return nil, err
 	}
 
 	if len(items) > 0 {
-		return false, errors.New("already attributed")
+		return nil, errors.New("already attributed")
```

Comment:
it doesn't mean that it's `attributed`, it means that cannot be attributed because its not empty.

Reply:
updated to return bit more info --> return nil, errors.New("bucket: " + string(req.BucketName) + " cannot be attributed")